pkg/networkservice/mechanisms/memif/client.go

```diff
@@ -41,13 +42,11 @@ const (
 	MECHANISM = memif.MECHANISM
 )
 
-type memifClient struct {
-	baseDir string
-}
+type memifClient struct{}
 
 // NewClient provides a NetworkServiceClient chain elements that support the memif Mechanism
 func NewClient(baseDir string) networkservice.NetworkServiceClient {
-	return &memifClient{baseDir: baseDir}
+	return &memifClient{}
```

Comment:
I think we should also remove the unused parameter `baseDir` from line 48

Reply:
Done :)